sormas-api/src/main/java/de/symeda/sormas/api/i18n/Captions.java

```diff
@@ -718,7 +718,7 @@
 	String dashboardComparedToPreviousPeriod = "dashboardComparedToPreviousPeriod";
 	String dashboardCompletedFollowUp = "dashboardCompletedFollowUp";
 	String dashboardCompletedFollowUpShort = "dashboardCompletedFollowUpShort";
-	String dashboardConfirmed = "dashboardConfirmed";
```

Comment:
Removing `dashboardConfirmed` breaks every translation. Just keeping it for cases would only fix the problem, without any need to change anything to translation files.

Reply:
While I agree, I am not sure if it will then be clear enough during translation. After all, just from seeing the "dashboardConfirmed" string the translator would not know if it's for cases or contacts, right?

Reply:
I understand your point, but breaking translations like this should not happen when fixing a defect like this. I think if questions arise what which captions are for, we need do clarify it somehow different. And think about how to make refactoring possible here at some point in the future.